lib/elixir/lib/inspect/algebra.ex

```diff
@@ -283,6 +311,23 @@ defmodule Inspect.Algebra do
     fold_doc(docs, &concat(&1, &2))
   end
 
+  @doc """
+  Colors a document if a the identifier has a color in the options.
+  """
+  @spec color(color_identifier, t, Inspect.Opts.t) :: doc_color
+  def color(identifier, doc, opts \\ %Inspect.Opts{}) when is_doc(doc) do
```

Comment:
Could we force the `opts` to be passed and remove the default argument? An empty `opts` will never use a color.